src/actix/api_key.rs

```diff
@@ -10,18 +10,19 @@ use futures_util::future::LocalBoxFuture;
 
 use crate::common::auth::AuthKeys;
 
+/// List of read-only POST request paths. List MUST be sorted.
```

Comment:
is it possible to have a debug_assert! somewhere to check that it is sorted?

Reply:
We can, but I created a [test](https://github.com/qdrant/qdrant/pull/3544/commits/45dd5e4a7cdc0986f6f14c9c4189452ff233a3ac) for it instead so our CI disallows breaking sorting.

I did try to check for sorting at compile time, but `is_sorted` and related functions are not `const` nor stabilized in the stdlib.